pkgs/os-specific/linux/cpupower/gui.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, stdenv, fetchFromGitHub
+, desktop-file-utils
+, gettext
+, glib
+, gobject-introspection
+, gtk3
+, hicolor-icon-theme
+, meson
+, ninja
+, pkg-config
+, python3
+, systemd
+, wrapGAppsHook
+}:
+
+let
+  # use pythonEnv, so that patchShebang fixup will choose this interpreter
+  pythonEnv = python3.withPackages(ps: with ps; [
+    pygobject3
+    dbus-python
+  ]);
+in
+
+stdenv.mkDerivation rec {
+  pname = "cpupower-gui";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "vagnum08";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1rrzlcmspill7qivv3bg8v810ppj4wxa9jcf2ddrjqcbq4lfxayk";
+  };
+
+  nativeBuildInputs = [
```

Comment:
u will need python3 as a nativeBuildInputs for the patchShebangs below

Reply:
added, thanks!